lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -724,6 +723,7 @@ def transaction_hash(result)
           'risk_data'                    => risk_data,
           'network_transaction_id'       => transaction.network_transaction_id || nil,
           'processor_response_code'      => response_code_from_result(result),
+          'additional_processor_response' => nil,
```

Comment:
Why do we need this here when additional_processor_response is already in line 627?

Reply:
There's no need to do this because in our side additional_processor_response will already be nil for successful response.